python-sdk/hatchet_sdk/hatchet.py

```diff
@@ -35,11 +35,12 @@ def inner(cls):
         
         return inner
 
-    def step(self, name : str='', timeout : str='', parents : List[str] = []):
+    def step(self, name : str='', timeout : str='', parents : List[str] = [], retries : int = 0):
```

Comment:
note for future: we should maybe start to consider using Pydantic for a configuration object here as config is starting to get a bit complex